src/todo_app/routers/users.py

```diff
@@ -3,19 +3,22 @@
 from typing import List, Annotated
 from sqlmodel import select
 
+
 from ..sqlmodel_orm.models.user_model import (
     Users,
     UsersCreate,
     UsersPublic,
     UsersUpdate,
 )
 from ..dependencies import SessionDep
+from ..internal.encrypt import hash_password
```

Comment:
Het is meer standaard om absolute imports te gebruiken in plaats van relatieve. Het zorgt ervoor dat de import altijd geldig blijft als de file verplaatst wordt en daarnaast is het ook duidelijker waar een bepaalde methode vandaan komt.

Reply:
Done